src/pybind/mgr/dashboard/frontend/src/app/core/error/error.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, HostListener, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute, NavigationEnd, Router, RouterEvent } from '@angular/router';
+
+import { Subscription } from 'rxjs';
+import { filter } from 'rxjs/operators';
+
+import { DocService } from '~/app/shared/services/doc.service';
+
+@Component({
+  selector: 'cd-error',
+  templateUrl: './error.component.html',
+  styleUrls: ['./error.component.scss']
+})
+export class ErrorComponent implements OnDestroy, OnInit {
+  header: string;
+  message: string;
+  section: string;
+  section_info: string;
+  icon: string;
+  docUrl: string;
+  routerSubscription: Subscription;
+
+  constructor(
+    public route: ActivatedRoute,
```

Comment:
It seems this variable is not used. If that is the case it should be removed.

Reply:
Done.